api/controlplane/kubeadm/v1beta2/kubeadm_control_plane_types.go

```diff
@@ -495,7 +495,6 @@ type KubeadmControlPlaneMachineTemplate struct {
 	// This field can be used e.g. to instruct the machine controller to include in the computation for Machine's ready
 	// computation a condition, managed by an external controllers, reporting the status of special software/hardware installed on the Machine.
 	//
-	// NOTE: This field is considered only for computing v1beta2 conditions.
```

Comment:
@fabriziopandini Not sure what the goal was. In https://github.com/kubernetes-sigs/cluster-api/pull/12091 we dropped this from some fields but not all. I think we should either keep it everywhere or drop it everywhere?